### PR TITLE
APID-208 - Do not send soap xml with replyTo, faultTo when they are r…

### PR DESCRIPTION
**Pull request by sivaprakashiv**
…esolved to empty string in the request

### CODE REVIEW

**Comment by sivaprakashiv** [comment]
Ignore the wrong ticket id in the commit desc

**Comment by sivaprakashiv** [comment]
This PR is based on the comment in the jira ticket [APID-208](https://jira.tools.tax.service.gov.uk/browse/APID-208) as follows: 

siva isikella as discussed with with Mat Evans can we make a change so:
If ReplyTo or FaultTo are an empty string then the XML elements in the SOAP need to be removed.

File: app/uk/gov/hmrc/apiplatformoutboundsoap/services/OutboundService.scala [hunk 1]
```diff
@@ -178,10 +178,10 @@ class OutboundService @Inject()(outboundConnector: OutboundConnector,
   }
 
   private def addOptionalAddressingHeaders(message: MessageRequest, wsaNs: OMNamespace, envelope: SOAPEnvelope): Unit = {
-    addWithAddressElementChildToSoapHeader(message.addressing.from, WSA_FROM, wsaNs, envelope)
+    addWithAddressElementChildToSoapHeader(x => x == x , message.addressing.from, WSA_FROM, wsaNs, envelope)
```
Comment (worthydolt) [inline app/uk/gov/hmrc/apiplatformoutboundsoap/services/OutboundService.scala]: This is clever but had me puzzled for a minute. Couldn't we just say `_ => true`?

**Review by worthydolt — COMMENTED** [review]
Just one small comment which I think would make the code clearer

File: app/uk/gov/hmrc/apiplatformoutboundsoap/services/OutboundService.scala (hunk 1 above)
Comment (sivaprakashiv) [reply app/uk/gov/hmrc/apiplatformoutboundsoap/services/OutboundService.scala]: yes, makes it much simpler. Done now

**Pull request merged**